src/library.js

```diff
@@ -460,7 +469,7 @@ mergeInto(LibraryManager.library, {
     {{{ makeSetValue('tmPtr', C_STRUCTS.tm.tm_yday, 'yday', 'i32') }}};
   },
 
-  _timegm_js__sig: 'ii',
+  _timegm_js__sig: 'pp',
```

Comment:
ditto

Reply:
Actually time_t is currently defined as `long` so is different under wasm64... I've fixed that now but might make a separate PR for that.